src/main/common/color.h

```diff
@@ -21,10 +21,12 @@
 typedef enum {
     RGB_RED = 0,
     RGB_GREEN,
-    RGB_BLUE
+    RGB_BLUE,
+    RGB_WHITE,
```

Comment:
The name of this is now muddled.  RGB means RED GREEN and BLUE.

Likely the prefixes now need to be RGBW.  `colorComponent_e` is still a perfectly valid name for RGB and RGBW however.